ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/collector/KsqlModuleType.java

```diff
@@ -17,5 +17,5 @@
 package io.confluent.ksql.version.metrics.collector;
 
 public enum KsqlModuleType {
-  LOCAL_CLI, REMOTE_CLI, EMBEDDED, SERVER;
+  CLI, SERVER;
```

Comment:
Could you please add unit test that verifies that MetricsValidation.isValidKsqlModuleType() returns true for every value of KsqlModuleType.values()?